http/httpclient/client.go

```diff
@@ -271,6 +271,9 @@ func (jc *HttpClient) UploadFileFromReader(reader io.Reader, url string, httpCli
 	if errorutils.CheckError(err) != nil {
 		return nil, nil, err
 	}
+	if resp != nil && resp.StatusCode != http.StatusCreated && resp.StatusCode != http.StatusOK {
+		return nil, nil, errorutils.CheckError(errors.New("Artifactory response: " + resp.Status))
```

Comment:
Since this is a generic http client, and not an http client used for Artifactory only, let's change - 
errors.New("Artifactory response: " + resp.Status)
-->
errors.New("Server response: " + resp.Status)